docs/pages/faq.mdx

```diff
@@ -142,13 +142,13 @@ supported for 9 months.
 
 ### Supported versions
 
-Here are the major versions of Teleport that we currently support:
+Here are the major versions of Teleport and their support windows:
 
-|Release|Release Date|Minimum `tsh` version|
-|---|---|---|
-|v11.0|October 24, 2022|v10.x.x|
-|v10.0|July 8, 2022|v9.x.x|
-|v9.0|March 11, 2022|v8.x.x|
+| Release | Release Date     | EOL          | Minimum `tsh` version|
+|---------|------------------|--------------|----------------------|
+| v11.0   | October 24, 2022 | July 2023    | v10.x.x              |
```

Comment:
@russjones is that the start of the month, or end of the month?

Reply:
End of the month, updated.